numpy/lib/histograms.py

```diff
@@ -262,6 +262,36 @@ def _hist_bin_auto(x, range):
         # limited variance, so we return a len dependent bw estimator
         return sturges_bw
 
+def _report_auto_method(x, range):
+    """
+    return the auto method used given an array
+
+    Parameters
+    ----------
+    x : array_like
+        Input data that is to be histogrammed, trimmed to range. May not
+        be empty.
+
+    Returns
+    -------
+    h : An estimate of the optimal bin width for the given data.
```

Comment:
not accurate - this returns either 'fd' or 'sturges' - whichever has the smaller number of bins